pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java

```diff
@@ -181,7 +185,7 @@ public void getOwnedNamespaces(@Suspended final AsyncResponse asyncResponse,
     }
 
     @POST
-    @Path("/configuration/{configName}/{configValue}")
+    @Path("/configuration/{configName}/{configValue}/{scope}")
```

Comment:
I'm afraid these changes are breaking changes.

Reply:
@tisonkun  I move parameters to queryParameter, PTAL again